chapters/background.tex

```diff
@@ -1,4 +1,14 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-
+Regarding soHappy, TesnorFlow \cite{tensorflow} appears to be a convenient fit when it comes to smile detection.
+TensorFlow is a open source machine learning library developed by Google.
+It supports a wide variety of programming languages, mainly targeting web and mobile applications.
+For mobile development TensorFlow provides a Lite version called TensorFlow Lite (TFLite) which can be easily included in a mobile app.
+The main advantage TFLite offers, is the ability to add a self trained model to detect smiles.
+Depending on the dataset a model is trained with, TFLite usually is able to detect emotions, for example happiness or sadness.
+However, soHappy only needs smile detection to comply with the core requirements.
+Therefore its model can be broken down to simply detect smiling/happiness, rather than having the whole emotion detection.
```

Comment:
I'd prefer a comma after "Therefore"

"Entire" sounds better than "whole" to me, personally. But it's up to your preference